examples/data/cleansed/metadata.yml

```diff
@@ -0,0 +1,61 @@
+scada:
+  time: time
+  id: id
+  WTUR_W: WTUR_W
+  WMET_HorWdSpd: WMET_HorWdSpd
+  WMET_HorWdDir: WMET_HorWdDir
+  status: status
+  WROT_BlPthAngVal: WROT_BlPthAngVal
+  WMET_EnvTmp: WMET_EnvTmp
+  energy: energy
+  frequency: 10T
+meter:
+  time: time
+  MMTR_SupWh: MMTR_SupWh
+  frequency: 10T
+tower:
+  time: time
+  id: id
+  frequency: 10T
+status:
+  time: time
+  id: id
+  status_id: status_id
+  status_code: status_code
+  status_text: status_text
+  frequency: 10T
+asset:
+  id: id
+  latitude: latitude
+  longitude: longitude
+  rated_power: rated_power
+  hub_height: hub_height
+  rotor_diameter: rotor_diameter
+  elevation: elevation
+  type: type
+curtail:
+  time: time
+  IAVL_ExtPwrDnWh: IAVL_ExtPwrDnWh
+  IAVL_DnWh: IAVL_DnWh
+  frequency: 10T
+reanalysis:
+  era5:
+    time: time
+    WMETR_HorWdSpd: WMETR_HorWdSpd
+    WMETR_HorWdSpdU: WMETR_HorWdSpdU
+    WMETR_HorWdSpdV: WMETR_HorWdSpdV
+    WMETR_HorWdDir: WMETR_HorWdDir
+    WMETR_EnvTmp: WMETR_EnvTmp
+    WMETR_AirDen: WMETR_AirDen
+    WMETR_EnvPres: WMETR_EnvPres
+    frequency: H
+  merra2:
+    time: time
+    WMETR_HorWdSpd: WMETR_HorWdSpd
+    WMETR_HorWdSpdU: WMETR_HorWdSpdU
+    WMETR_HorWdSpdV: WMETR_HorWdSpdV
+    WMETR_HorWdDir: WMETR_HorWdDir
+    WMETR_EnvTmp: WMETR_EnvTmp
+    WMETR_AirDen: WMETR_AirDen
+    WMETR_EnvPres: WMETR_EnvPres
+    frequency: H
```

Comment:
what is this cleansed metadata file used for?

Reply:
This is used for the examples. I just made it so that if the data is output (as in the 00 example), then the other examples can rely on the prepared data rather than reloading the data and rerunning all the preparation steps. This addition makes it quicker to get an example loaded,  which is nice if someone is running them on their own in Binder or locally.